lib/utils.js

```diff
@@ -19,6 +19,7 @@ var etag = require('etag');
 var proxyaddr = require('proxy-addr');
 var qs = require('qs');
 var querystring = require('querystring');
+var flatten = require('array-flatten');
```

Comment:
Actually, nevermind, looks like I already merged this commit into the 4.13 branch :) Looks like you need take no action at all :D